src/components/item/publish/PublicationAttributeContainer.tsx

```diff
@@ -0,0 +1,144 @@
+import { MouseEvent } from 'react';
+
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
+import WarningIcon from '@mui/icons-material/Warning';
+import { Box, Stack, SxProps, Tooltip, Typography } from '@mui/material';
+
+import { theme } from '@graasp/ui';
+
+import { WARNING_COLOR } from '@/config/constants';
+import {
+  buildPublishAttrContainer,
+  buildPublishAttrEmptyContainer,
+  buildPublishTitleAction,
+  buildPublishWarningIcon,
+} from '@/config/selectors';
+
+import ContentLoader from '../../common/ContentLoader';
+
+const clickableBoxSx: SxProps = {
+  cursor: 'pointer',
+  ':hover': {
+    boxShadow: '0 1px 3px rgba(0, 0, 0, 0.2)',
+    transform: 'scale(1.01)',
+  },
+  ':active': {
+    transform: 'scale(1.0)',
+  },
+};
+
+const CONTAINER_BORDER = {
+  radius: '8px',
+  border: '1px #ddd solid',
+};
+
+type Props = {
+  dataTestId: string;
+  title: string;
+  titleIcon?: JSX.Element;
+  titleActionBtn: JSX.Element;
+  content?: JSX.Element | JSX.Element[];
+  isLoading?: boolean;
+  emptyDataMessage: string;
+  attributeDescription: string;
+  onEmptyClick: () => void;
+};
+
+export const PublicationAttributeContainer = ({
+  dataTestId,
+  title,
+  titleIcon,
+  titleActionBtn,
+  content,
+  emptyDataMessage,
+  attributeDescription,
+  isLoading = false,
+  onEmptyClick,
+}: Props): JSX.Element => {
+  const hasNoData =
+    content === undefined || (Array.isArray(content) && content.length === 0);
+  const hasData = !hasNoData;
+
+  const buildEmptyPlaceHolder = () => (
+    <Stack
+      direction="column"
+      alignItems="center"
+      justifyContent="center"
+      flexGrow={1}
+      data-cy={buildPublishAttrEmptyContainer(dataTestId)}
+    >
+      <AddCircleOutlineIcon
+        htmlColor={theme.palette.primary.main}
+        sx={{ fontSize: 40 }}
+      />
+      <Typography
+        variant="body1"
+        color={theme.palette.text.secondary}
+        textAlign="center"
+      >
+        {emptyDataMessage}
+      </Typography>
+    </Stack>
+  );
+
+  const handleOnContainerClick = () => {
+    if (hasNoData) {
+      onEmptyClick();
+    }
+  };
+
+  // Stop propagation to prevent executing handleOnContainerClick.
+  const handleTitleIconClick = (e: MouseEvent) => e.stopPropagation();
+
+  return (
+    <ContentLoader isLoading={isLoading}>
+      <Stack
+        data-cy={buildPublishAttrContainer(dataTestId)}
+        border={CONTAINER_BORDER.border}
+        borderRadius={CONTAINER_BORDER.radius}
+        padding={2}
+        onClick={handleOnContainerClick}
+        sx={hasNoData ? clickableBoxSx : {}}
+        flexBasis="100%"
+        spacing={1}
+      >
+        <Stack
+          direction="row"
+          alignItems="center"
+          justifyContent="space-between"
+        >
+          <Stack direction="row" alignItems="center">
+            <Stack
+              direction="row"
+              alignItems="center"
+              onClick={handleTitleIconClick}
```

Comment:
Is it necessary? It think it's okay to click on the title to open modal?

Reply:
It is used to avoid opening the modal when clicking on the help button for the License. I can move it down on the title icon.